### PR TITLE
Prototypal inheritance

### PR DESCRIPTION
**Pull request by mahdiHashemi14**
I didn't know how to translate the comment in [line 169](https://github.com/javascript-tutorial/fa.javascript.info/blob/9243586de6000ad116e00751678545bc6551bd06/1-js/08-prototypes/01-prototype-inheritance/article.md?plain=1#L169). Best to take a look.

Please review and comment/commit of anything is wrong.

### CODE REVIEW

**Comment by mahdiHashemi14** [comment]
I didn't know how to translate [line 169](https://github.com/javascript-tutorial/fa.javascript.info/blob/9243586de6000ad116e00751678545bc6551bd06/1-js/08-prototypes/01-prototype-inheritance/article.md?plain=1#L169). Best to take a look.

File: 1-js/08-prototypes/01-prototype-inheritance/article.md [hunk 1]
```diff
@@ -173,13 +173,13 @@ rabbit.walk = function() {
 rabbit.walk(); // Rabbit! Bounce-bounce!
```
Comment (mahdyar) [inline 1-js/08-prototypes/01-prototype-inheritance/article.md:173]:
```suggestion
rabbit.walk(); // خرگوش! بپر-بپر!
```

**Review by mahdyar — CHANGES_REQUESTED** [review]

**Comment by mahdiHashemi14** [comment]
Thank you @mahdyar 
/done

**Review by mahdyar — APPROVED** [review]
Thank you, @mahdiHashemi14!

**Pull request merged**